rpc/js/src/dial.ts

```diff
@@ -458,44 +468,47 @@ export async function dialWebRTC(
     let exchangeDone = false;
     if (!webrtcOpts?.disableTrickleICE) {
       // set up offer
-      const offerDesc = await pc.createOffer();
+      const offerDesc = await pc.createOffer({});
```

Comment:
is this change significant to the logic or just a linting recommendation?

Reply:
It's significant to the logic -- the react-native version need the empty dict